BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleAccountsViewController.m

```diff
@@ -40,7 +40,7 @@ - (void)viewDidLoad {
         BOXContentClient *client = [BOXContentClient clientForUser:user];
         if (   ([client.session isKindOfClass:[BOXOAuth2Session class]] && !self.isAppUsers)
             || ([client.session isKindOfClass:[BOXAppUserSession class]] && self.isAppUsers)) {
-            [client.urlSessionManager setUpWithDefaultDelegate:((id<BOXURLSessionManagerDelegate>)[[UIApplication sharedApplication] delegate])];
+            [client setUpToSupportBackgroundTasksWithDefaultDelegate:((id<BOXURLSessionManagerDelegate>)[[UIApplication sharedApplication] delegate])];
```

Comment:
We might need to do this logic a bit differently if it is only expected to be called once (it's currently being called here for a new client, or below for retrieving an existing client - both of those things can happen a bunch of times in the life of the sample app).